docs/src/content/docs/de/guides/pages.mdx

```diff
@@ -0,0 +1,178 @@
+Um das Starlight-Layout in benutzerdefinierten Seiten zu verwenden, umhüllst du deinen Seiteninhalt mit der Komponente `<StarlightPage />`.
+Das kann hilfreich sein, wenn du Inhalte dynamisch generierst, aber trotzdem das Starlight-Design verwenden willst.
+
+```astro
+---
+// src/pages/custom-page/example.astro
+import StarlightPage from '@astrojs/starlight/components/StarlightPage.astro';
+import CustomComponent from './CustomComponent.astro';
+---
+
+<StarlightPage frontmatter={{ title: 'Meine eigene Seite' }}>
+	<p>
+		Dies ist eine benutzerdefinierte Seite mit einer benutzerdefinierten
+		Komponente:
+	</p>
+	<CustomComponent />
+</StarlightPage>
+```
+
+#### Eigenschaften
+
+Die Komponente `<StarlightPage />` akzeptiert die folgenden Eigenschaften.
+
+##### `frontmatter` (erforderlich)
+
+**Typ:** `StarlightPageFrontmatter`
+
+Setzt die [Frontmatter Eigenschaften](/de/reference/frontmatter/) für diese Seite, ähnlich wie Frontmatter in Markdown-Seiten.
+Die Eigenschaft [`title`](/de/reference/frontmatter/#title-erforderlich) ist erforderlich und alle anderen Eigenschaften sind optional.
+
+Die folgenden Eigenschaften unterscheiden sich von Markdown frontmatter:
+
+- Die Eigenschaft [`slug`](/de/reference/frontmatter/#slug) wird nicht unterstützt und wird automatisch anhand der URL der benutzerdefinierten Seite gesetzt.
+- Die Option [`editUrl`](/de/reference/frontmatter/#editurl) erfordert eine URL, um einen Bearbeitungslink anzuzeigen.
+- Die Frontmatter-Eigenschaft [`sidebar`](/de/reference/frontmatter/#sidebar), mit der du festlegen kannst, wie die Seite in [autogenerierten Linkgruppen](/de/reference/configuration/#sidebar) erscheint, ist nicht verfügbar. Seiten, die die Komponente `<StarlightPage />` verwenden, sind nicht Teil einer Sammlung und können nicht zu einer automatisch erzeugten Seitenleistengruppe hinzugefügt werden.
+- Die Option [`draft`](/de/reference/frontmatter/#draft) zeigt nur einen [Hinweis](/de/reference/overrides/#draftcontentnotice) an, dass die Seite ein Entwurf ist, schließt sie aber nicht automatisch von der Produktion aus.
+
+##### `Sidebar`
+
+**Typ:** [`SidebarItem[]`](/de/reference/configuration/#sidebaritem)  
+**Voreinstellung:** die Seitenleiste, die auf der Grundlage der [globalen `Sidebar`-Konfiguration](/de/reference/configuration/#sidebar) erzeugt wird
+
+Legt eine benutzerdefinierte Navigationsleiste für diese Seite fest.
+Wenn sie nicht gesetzt wird, verwendet die Seite die globale Standard-Seitenleiste.
+
+Die folgende Seite ersetzt zum Beispiel die Standard-Seitenleiste durch einen Link zur Homepage und eine Gruppe von Links zu verschiedenen anderen benutzerdefinierten Seiten.
+
+```astro {3-13}
+<StarlightPage
+	frontmatter={{ title: 'Orion' }}
+	sidebar={[
+		{ label: 'Home', link: '/' },
+		{
+			label: 'Constellations',
+			items: [
+				{ label: 'Andromeda', link: '/andromeda/' },
+				{ label: 'Orion', link: '/orion/' },
+				{ label: 'Ursa Minor', link: '/ursa-minor/', Abzeichen: 'Stub' },
```

Comment:
```suggestion
				{ label: 'Ursa Minor', link: '/ursa-minor/', badge: 'Platzhalter' },
```

The property name should not be translated and I think we used "Platzhalter" in https://starlight.astro.build/de/guides/sidebar/#abzeichenvarianten